config.go

```diff
@@ -1,6 +1,23 @@
 package config
 
-const Version = "1.0.0"
+const (
+	Version               = "1.0.0"
+	GithubWritePermission = "write"
+
+	// Constants to be used as keys in the config files
+
+	BaseServerURL          = "baseServerUrl"
+	TelemetryEnabled       = "telemetryEnabled"
+	SDKClassName           = "sdkClassName"
+	SingleTagPerOp         = "singleTagPerOp"
+	TagNamespacingDisabled = "tagNamespacingDisabled"
+	Languages              = "languages"
+	Mode                   = "mode"
+	GithubAccessToken      = "github_access_token"
+	SpeakeasyApiKey        = "speakeasy_api_key"
+	OpenAPIDocAuthHeader   = "openapi_doc_auth_header"
+	OpenAPIDocAuthToken    = "openapi_doc_auth_token"
```

Comment:
This is _not_ another list that we have to append to every time we introduce a new config field. Just a place for specifying constants when accessing config map keys (ex. `config.Generation.Fields[sdkGenConfig.TagNamespacingDisabled]` in openapi-generation)